src/inout.f90

```diff
@@ -124,26 +152,33 @@ SUBROUTINE Input(IErr)
   ! check the parameters for validity
   
   IF(IWriteFlag.GE.2) THEN
-     PRINT*,"ISeed         = ", ISeed
-     PRINT*,"NSeed         = ", NSeed
-     PRINT*,"Dim           = ", Dim
-     PRINT*,"Nx            = ", Nx
-     PRINT*,"IBCFlag       = ", IBCFlag
-     PRINT*,"IRNGFlag      = ", IRNGFlag
-     PRINT*,"IKeepFlag     = ", IKeepFlag
-     PRINT*,"IWriteFlag    = ", IWriteFlag
-     PRINT*,"IStateFlag    = ", IStateFlag
-     PRINT*,"Width0        = ", Width0
-     PRINT*,"Width1        = ", Width1
-     PRINT*,"dWidth        = ", dWidth
-     PRINT*,"CubeConPot0   = ", CubeConPot0
-     PRINT*,"CubeConPot1   = ", CubeConPot1
-     PRINT*,"dCubeConPot   = ", dCubeConPot
-     PRINT*,"CubeDis0      = ", CubeDis0
-     PRINT*,"CubeDis1      = ", CubeDis1
-     PRINT*,"dCubeDis      = ", dCubeDis
-     PRINT*,"LiebConPot= ", LiebConPot
-     PRINT*,"LiebDis       = ", LiebDis     
+     PRINT*,"ISeed          = ", ISeed
+     PRINT*,"NSeed          = ", NSeed
+     PRINT*,"Dim            = ", Dim
+     PRINT*,"Nx             = ", Nx
+     PRINT*,"IBCFlag        = ", IBCFlag
+     PRINT*,"IRNGFlag       = ", IRNGFlag
+     PRINT*,"IKeepFlag      = ", IKeepFlag
+     PRINT*,"IWriteFlag     = ", IWriteFlag
+     PRINT*,"IStateFlag     = ", IStateFlag
```

Comment:
Sentence " PRINT*, "IFluxFlag      = ", IFluxFlag " should be added here.

Reply:
Done.